CMakeLists.txt

```diff
@@ -138,6 +138,7 @@ endif()
 add_subdirectory(ElfUtils)
 add_subdirectory(OrbitBase)
 add_subdirectory(OrbitCaptureClient)
+add_subdirectory(OrbitClientData)
```

Comment:
I realised it would make more sense to leave the comment here instead of the main conversation: wondering if it would make sense to use the recently created _OrbitClientModel_ to allocate the data files instead of creating a new _OrbitClientData_ project. I feel that the purpose of both projects is similar

Reply:
See my other comment, it is basically to avoid cyclic dependencies between different components